### ISSUE TITLE
want to get some advice for training.

### ISSUE THREAD

**Issue by DavideHe** [d-li14/mobilenetv2.pytorch#26]
mobilenetv2 1.0 224
epochs: 200
bacth size: 512
lr-decay: cos
lr : 0.2 
wd: 4e-5
wamps up: 0-0.4  first 5 epochs
dropout : 0.2

top1 : 0.685
but In your md_file ,it is 0.722
is something  wrong in my training?
